plan.sh

```diff
@@ -1,7 +1,7 @@
 pkg_name=cmake
 pkg_origin=core
-_base_version=3.20.0-rc4
-pkg_version=${_base_version}
+_base_version=3.20
+pkg_version=${_base_version}.0-rc4
```

Comment:
I don't think we want the RC. There should be a 3.20.1 (see https://cmake.org/download/)

Reply:
Bumped to [3.20.3 on instance](https://github.com/chef-base-plans/cmake/pull/6/commits/e90f4513e042a92ee1a32a2ea7fd282b48f5ce3f)